XCUITests/SaveLoginsTests.swift

```diff
@@ -44,21 +103,28 @@ class SaveLoginTest: BaseTestCase {
         app.buttons["SaveLoginPrompt.dontSaveButton"].tap()
         // There should not be any login saved
         openLoginsSettings()
-        XCTAssertFalse(app.tables.cells[savedLoginEntry].exists)
-        XCTAssertEqual(app.tables["Login List"].cells.count, 0)
+        XCTAssertFalse(app.staticTexts[domain].exists)
+        XCTAssertFalse(app.staticTexts[domainLogin].exists)
+        XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsEmptyLoginsList)
     }
 
     // Smoketest
     func testSavedLoginSelectUnselect() {
-        saveLogin()
+        saveLogin(givenUrl: testLoginPage)
+        navigator.goto(SettingsScreen)
         openLoginsSettings()
-        XCTAssertTrue(app.tables.cells[savedLoginEntry].exists)
-
+        XCTAssertTrue(app.staticTexts[domain].exists)
+        XCTAssertTrue(app.staticTexts[domainLogin].exists)
+        //app.staticTexts[domain].tap()
+        print(app.debugDescription)
         app.buttons["Edit"].tap()
+        print(app.debugDescription)
+        // This doesn't seem to work after refactor
```

Comment:
Right, I have checked this and looks like although the ID for the image is there, see [code here](https://github.com/mozilla-mobile/firefox-ios/blob/75f96daac75dd1c8a6583a481ec266acf0100d41/Client/Frontend/Login%20Management/LoginTableViewCell.swift#L92), it is not displayed, worth checking with devs why...

Reply:
Oh wait...I can see that ids again...
Image, 0x6000025be3c0, {{-33.0, 333.5}, {22.0, 22.0}}, identifier: 'loginUnselected'...